src/lamp_py/ingestion/convert_gtfs_rt.py

```diff
@@ -97,6 +98,8 @@ def __init__(
             self.detail = RtBusVehicleDetail()
         elif config_type == ConfigType.BUS_TRIP_UPDATES:
             self.detail = RtBusTripDetail()
+        elif config_type == ConfigType.LIGHT_RAIL:
+            raise IgnoreIngestion("Ignore LIGHT_RAIL files")
```

Comment:
This seemed like the least intrusive way to handle skipping these files in GTFS-RT processing. Specific exception will trigger the ignoring of this file type.